rrfs-test/validated_yamls/templates/obtype_config/msonet_airTemperature_188.yaml

```diff
@@ -76,35 +76,109 @@
              is_in: 188
            action:
              name: inflate error
-             #inflation factor: 1.09757
              inflation variable:
                name: ObsFunction/ObsErrorFactorPressureCheck
```

Comment:
I couldn't find any description of the ObsErrorFactorPressureCheck function. Could you explain what it does and how to use it? When I tried using it in my experiment, I encountered an assertion failure because the pressures were infinitely large, resulting in NaN values for the log pressures.

Reply:
@spanNOAA, yes, I've encountered this several times. What is most likely happening is you have observations outside the domain. You can't use an online domain check here since you're not able to drop those obs from memory and the ObsErrorFactorPressureCheck doesn't know about the QC flags set by the online check. You must use the offline domain check for now. If that doesn't fix your problem let me know. 

As for documentation:
https://github.com/JCSDA-internal/ufo/blob/develop/src/ufo/filters/obsfunctions/ObsErrorFactorPressureCheck.h
https://github.com/JCSDA-internal/ufo/blob/develop/src/ufo/filters/obsfunctions/ObsErrorFactorPressureCheck.cc

It basically mimics some ob error inflation done in GSI if the ob height is within the model levels or below it.

There isn't any such documentation on the JEDI doc webpage, but one of the tricks I use to find these things is to head to the top of my RDASApp build and then
```
cd RDASApp/sorc
grep -ir --include="*.yaml" <search string> .
```
and then you can browse yamls and if you find a function like this one, you can easily find it in the code by
```
find . -name "ObsErrorFactorPressureCheck*"
```